### ISSUE TITLE
`Mail for domain enabled` should be renamed to `Accept mail for domain`

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#930]
| SYSTEM INFORMATION ||
|----------|----------|
|  OS type and version |  Ubuntu Linux 22.04.5              |
|  Usermin version     |  2.102                             |
|  Virtualmin version  |  7.20.2 Pro                        |
|  Theme version       |  21.20.7                           |
|  Apache version      |  2.4.52                            |
|  Package updates     |  10 package updates are available  |

### the issue

`Mail for domain enabled` implies postfix and dovecot (i.e. in and out) but in actuality this option only controls the receiving of emails so I think the name is wrong.

on the page `Virtualmin --> Edit Virtual Server --> Mail for domain enabled`

![image](https://github.com/user-attachments/assets/a20846dc-f4cb-4764-b181-d2d9dd0d97a1)

on the page `Virtualmin --> Create Virtual Server --> Accept mail for domain`

![image](https://github.com/user-attachments/assets/85bae517-5ff5-4132-83ab-7a15292ce2d9)

on the page `Virtualmin --> System Settings --> Account Plans --> Allowed available features --> Mail for domain`

![image](https://github.com/user-attachments/assets/0a58f97e-b935-4a8c-90c5-cb3fd648112f)

### proposed solution

Rename `Mail for domain enabled` to `Accept mail for domain` on the pages
- `Virtualmin --> Edit Virtual Server`
- `Virtualmin --> System Settings --> Account Plans --> Allowed available features `

**Comment by iliajie** [comment]
Hello,

> Rename Mail for domain enabled to Accept mail for domain

I don’t think this is a good change because “Mail for domain” is more of a cumulative term that includes spam and virus filtering, DKIM, and other related features.

**Comment by shoulders** [comment]
But it also implies outgoings.

Maybe there is another term that could replace these 3 terms and unify them.

They definitely should all be called the same though.

Incoming mail for domain
Incoming mail
Dovecot
Receive email

**Comment by iliajie** [comment]
I don’t think it needs to be changed, as “Mail for domain” is perfectly suitable and comprehensive enough in my view.

**Comment by shoulders** [comment]
I just noticed there are 3 variants of this single option's name:

- `Mail for domain enabled`
- `Accept mail for domain`
- `mail for domain`

Whichever name they have, I think these options should be named the same.

thanks

**Comment by iliajie** [comment]
> Whichever name they have, I think these options should be named the same.

Sure! Which pages are you referring to?

**Comment by shoulders** [comment]
### `Virtualmin --> Edit Virtual Server --> Mail for domain enabled`

![image](https://github.com/user-attachments/assets/a20846dc-f4cb-4764-b181-d2d9dd0d97a1)

### `Virtualmin --> Create Virtual Server --> Accept mail for domain`

![image](https://github.com/user-attachments/assets/85bae517-5ff5-4132-83ab-7a15292ce2d9)

### `Virtualmin --> System Settings --> Account Plans --> Allowed available features --> Mail for domain`

![image](https://github.com/user-attachments/assets/0a58f97e-b935-4a8c-90c5-cb3fd648112f)

**Comment by iliajie** [comment]
> Virtualmin --> Create Virtual Server --> Accept mail for domain

Good point! I have fixed that to be named "Enable mail for domain".

**Comment by shoulders** [comment]
Cam you look at unifying these feature names between these 3 areas. such as removing the word enable and setup.

These options all do the same thing so should be called the same.

tar

**Comment by iliajie** [comment]
> These options all do the same thing so should be called the same

Agreed! Fixed now! Thanks for the heads up!
